awx/main/tests/unit/utils/test_receptor.py

```diff
@@ -3,7 +3,7 @@
 
 def test_file_cleanup_scenario():
     args = _convert_args_to_cli({'exclude_strings': ['awx_423_', 'awx_582_'], 'file_pattern': '/tmp/awx_*_*'})
-    assert ' '.join(args) == 'cleanup --exclude-strings=awx_423_ awx_582_ --file-pattern=/tmp/awx_*_*'
+    assert ' '.join(args) == 'cleanup --exclude-strings="awx_423_ awx_582_" --file-pattern=/tmp/awx_*_*'
```

Comment:
As per the backlinked issue, the resultant meaning of this was to exclude any directory that has the `"awx_423_ awx_582_"` literal in its name. That is, in practical terms, excluding no directories.